cmd/kubeadm/app/cmd/phases/util.go

```diff
@@ -30,3 +53,7 @@ func SetKubernetesVersion(cfg *kubeadmapiv1beta1.ClusterConfiguration) {
 	}
 	cfg.KubernetesVersion = version.Get().String()
 }
+
+func getPhaseDescription(component string) string {
```

Comment:
i'm pretty sure this will cause a build problem given `getPhaseDescription` is already declared in the same package?
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/phases/controlplane.go

we should remove it from the above file if it is now in utils.go

Reply:
In this PR I'm removing it from `controlplane.go` (renamed to `controlplaneinit.go`) and moved it to `util.go`, so there's no problem with that.

That being said, maybe `util.go` is too generic and we might want a `controlplaneutil.go` even in the same package just for better organization?

Reply:
> In this PR I'm removing it from controlplane.go
i guess i missed that.

`util.go` seems OK for now. we can do some of these refactors later.